_rules/SC2-4-2-page-has-title.md

```diff
@@ -22,19 +22,19 @@ authors:
 
 ### Applicability
 
-The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.
+The rule applies to any page where the [document element](https://www.w3.org/TR/dom/#document-element) is an `html` element, and where the page is not embedded in another page.
 
-**Note**: Pages may be embedded inside other pages through elements such as iframes and object elements.
+**Note**: Pages may be embedded inside other pages through elements such as `iframe` and `object` elements.
 
 ### Expectation 1
 
-The page contains at least one `title` element.
+The [root](https://www.w3.org/TR/dom41/#concept-tree-root) element has at least one [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) that is an HTML `title` element.
```

Comment:
Writing considerations  :-) In the #380 you write `<title>` and in that rule `title` without angle brackets.

Reply:
You are right. I have removed the brackets in the other rule, to align with the general style used in Auto-WCAG...